docs/docs-beta/docs/dagster-plus/deployment/serverless/transition-hybrid.md

```diff
@@ -4,4 +4,35 @@ displayed_sidebar: "dagsterPlus"
 sidebar_position: 50
 ---
 
-# Transition from Serverless to Hybrid
\ No newline at end of file
+After using a Dagster+ Serverless deployment, you may decide that you want to switch to a Hybrid deployment. A Hybrid deployment lets you use your own infrastructure to execute your code. You can switch from Serverless to Hybrid without loosing any of your execution history or metadata.
+
+<details>
+  <summary>Prerequisites</summary>
+
+To follow the steps in this guide, you'll need:
+
+- **Organization Admin** permissions in your Dagster+ account.
+
+</details>
+
+## Step 1: Deactivate your Serverless agent
+To deactivate the Serverless agent, navigate to the **Deployment -> Agents** page. Click the drop down arrow on the right of the page and select **Switch to hybrid**. It may take a few minutes for the agent to deactivate and be removed from the list of agents.
+
+PRODUCT NOTE - this arrow drop down is pretty small and easy to confuse with the one in the row for the agent
```

Comment:
maybe a screen shot would help. i can also log this as a task in product feedback?